packages/one-app-runner/src/startApp.js

```diff
@@ -138,7 +138,7 @@ const generateDebug = (port, useDebug) => (useDebug ? `--inspect=0.0.0.0:${port}
 // So we have to remove those flags if the one-app version is less than 5.13.0
 // 5.13.0 is when node 16 was introduced.
 const generateNodeFlags = (appVersion) => {
-  if (semver.intersects(appVersion, '^5.13.0', { includePrerelease: true })) {
+  if (appVersion === 'latest' ? true : semver.intersects(appVersion, '^5.13.0', { includePrerelease: true })) {
```

Comment:
```suggestion
  if (appVersion === 'latest' || semver.intersects(appVersion, '^5.13.0', { includePrerelease: true })) {
```

We could avoid the ternary, but its not a blocker

Reply:
actually, now that I think about it, the reason I added ternary was because I kept getting this error with the or operator:

 Invalid comparator: latest , 

it seems that semver.intersects doesnt like "latest". So, reverting back to my original code that avoids semver when latest is specified.